ext/kv/proto/datapath.proto

```diff
@@ -1,97 +0,0 @@
-// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
```

Comment:
I think this file should be kept in the same place as `remote.rs`. Currently the `remote` backend still lives in the Deno repo, but the datapath proto lives in `denokv`.

Reply:
If we leave it here we'll have two "canonical" versions of the proto file lying around. I think this is bad.

We should either:
- do what I did here
- move the remote backend into `denokv` repo. WDYT?

Reply:
If we leave it here we'll have two "canonical" versions of the proto file lying around. I think this is bad.

We should either:
- do what I did here
- move the remote backend into `denokv` repo. WDYT?

Reply:
Any downside to moving the remote backend to `denokv` repo? Seems like a nice property to have all KV implementations in `denokv` repo.

Reply:
I can - it's a bit annoying to wire through all of the permissions things, but I can definitely do it.

Reply:
remote backend has been moved out